apis/r/tools/get_tarball.R

```diff
@@ -21,6 +21,7 @@ if (isMac) {
       stop("Unsupported Mac architecture. Please have TileDB Core installed locally.")
     }
 } else if (isLinux) {
+    # TODO: LOOK AT THIS
```

Comment:
Right on. The Linux side, as far as pre-made blobs are concerned, only knows x86_64 per the following line.

(Same thing is happening with Windows now where R build start for arm64, but not a concern for us yet as we do not do Windows, and also do not ship to CRAN).

Theses nested build scripts are not great but "evolved".  You can test a little in, say, a container by running each of one of them (down from the initial call from configure) and see where things are after each steps.